src/libraries/System.Memory.Data/ref/System.Memory.Data.cs

```diff
@@ -10,24 +10,33 @@ namespace System
     public partial class BinaryData
     {
         public BinaryData(byte[] data) { }
+        public BinaryData(byte[] data, string? mediaType) { }
         [System.Diagnostics.CodeAnalysis.RequiresDynamicCodeAttribute("JSON serialization and deserialization might require types that cannot be statically analyzed and might need runtime code generation.")]
         [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("JSON serialization and deserialization might require types that cannot be statically analyzed.")]
         public BinaryData(object? jsonSerializable, System.Text.Json.JsonSerializerOptions? options = null, System.Type? type = null) { }
         public BinaryData(object? jsonSerializable, System.Text.Json.Serialization.JsonSerializerContext context, System.Type? type = null) { }
         public BinaryData(System.ReadOnlyMemory<byte> data) { }
+        public BinaryData(System.ReadOnlyMemory<byte> data, string? mediaType) { }
         public BinaryData(string data) { }
+        public BinaryData(string data, string? mediaType) { }
         public static System.BinaryData Empty { get { throw null; } }
+        public string? MediaType { get { throw null; } }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override bool Equals([System.Diagnostics.CodeAnalysis.NotNullWhenAttribute(true)] object? obj) { throw null; }
         public static System.BinaryData FromBytes(byte[] data) { throw null; }
+        public static System.BinaryData FromBytes(byte[] data, string? mediaType) { throw null; }
         public static System.BinaryData FromBytes(System.ReadOnlyMemory<byte> data) { throw null; }
+        public static System.BinaryData FromBytes(System.ReadOnlyMemory<byte> data, string? mediaType) { throw null; }
```

Comment:
I don't see this overload here:
https://github.com/dotnet/runtime/issues/87068#issuecomment-1583142991

Reply:
I suspect this was just an oversight in the API listing. /cc @bartonjs @KrzysztofCwalina

Reply:
yeah, I think it's an oversight